lms/djangoapps/courseware/tests/test_tabs.py

```diff
@@ -767,6 +770,26 @@ def test_static_tab(self):
 
 
 @attr(shard=1)
+class CourseInfoTabTestCase(TabTestCase):
+    """Test cases for the course info tab."""
+    def setUp(self):
+        self.user = self.create_mock_user()
+        self.request = get_mock_request(self.user)
+
+    def test_default_tab(self):
+        # Verify that the course info tab is the first tab
+        tabs = get_course_tab_list(self.request, self.course)
+        self.assertEqual(tabs[0].type, 'course_info')
+
+    @patch('waffle.flag_is_active')
+    def test_default_tab_for_new_course_experience(self, patched_flag_is_active):
+        # Verify that the unified course experience hides the course info tab
+        patched_flag_is_active.return_value = True
```

Comment:
@andy-armstrong 👍   Thank you!
There is also this method of overriding, unless it didn't work here for some reason:
http://waffle.readthedocs.io/en/v0.11/testing/automated.html
No need to change the code.